packages/api/data/template/listings.json

```diff
@@ -293,4 +293,4 @@
 {"title":"Luminous apartment in the heart of the city","city":"Montral","bedrooms":3,"bathrooms":3,"amenities":["garden","heating","fireplace"],"description":"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.","img":"pic-purple.png","slug":{"$numberLong":"255193"}}
 {"title":"Luxurious apartment by the park","city":"Houston","bedrooms":2,"bathrooms":1,"amenities":["fireplace","heating","garden"],"description":"Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur.","img":"pic-pink.png","slug":{"$numberLong":"608072"}}
 {"title":"Bright apartment close to the park","city":"Sidney","bedrooms":3,"bathrooms":3,"amenities":["balcony","elevator","sauna"],"description":"Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus.","img":"pic-yellowgreen.png","slug":{"$numberLong":"586853"}}
-{"title":"Cozy apartment in the heart of the city","city":"Beijing","bedrooms":5,"bathrooms":2,"amenities":["fireplace","heating","wi-fi"],"description":"Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur.","img":"pic-pink.png","slug":{"$numberLong":"629909"}}
```

Comment:
I'm not really sure about the intended change in this file. Can you please give us some pointers? Thank you!

Reply:
I was trying to see where the ratings were coming from😂. No change was intended in this file

The line 296 seems identical to mine. Ive tried deleting line 297 but its still being highlighted in red.

I literally compared both lines - they seem identical

![image](https://github.com/Azure-Samples/contoso-real-estate/assets/135356007/806ce525-cc61-458e-97be-404ab41b9696)